pkg/scheduler/core/generic_scheduler.go

```diff
@@ -278,7 +279,38 @@ func (g *genericScheduler) findNodesThatFitPod(ctx context.Context, prof *profil
 
 	}
 
-	feasibleNodes, err := g.findNodesThatPassFilters(ctx, prof, state, pod, filteredNodesStatuses)
+	var nodes []*framework.NodeInfo
+	// The "NominatedNodeName" field is set due to the preemption in the previous scheduling cycle, the node
+	// most likely is still the only candidate for the preemptor pod.
+	// Have a try on this node directly when the option tryNominatedNodeFirst is set.
+	tryNominatedNodeFirst := g.tryNominatedNodeFirst && len(pod.Status.NominatedNodeName) > 0
```

Comment:
by default, this option is set to false but would love to hear from you about this option and the default value.